Automation/S3/delete_s3_buckets.py

```diff
@@ -0,0 +1,194 @@
+                    # Prepare just in case we want to delete the bucket
+                    bucket_object_list_prepared_for_deletion.append({
+                        'Key': bucket_object
+                    })
+                    print(bucket_object)
+            
+            # Prompt user for response
+            delete_bucket_response = ""
+            while delete_bucket_response == "":
+                deletion_prompt_response = input(Fore.YELLOW + "Delete bucket: (y/N/exit)")
+                if deletion_prompt_response == "y":
+                    delete_bucket_response = "True"
+                elif deletion_prompt_response == "N":
+                    delete_bucket_response = "False"
+                elif deletion_prompt_response == "exit":
+                    delete_bucket_response = "Exit"
+
+            
+            if delete_bucket_response == "True":
+                deletion_information = {
+                    'BucketName': bucket_name,
+                    'Objects': bucket_object_list_prepared_for_deletion
+                }
+                buckets_to_delete.append(deletion_information)
+                print(Fore.GREEN + "Object scheduled for deletion!")
+            elif delete_bucket_response == "Exit":
+                break
+
+            print()
+        
+        print()
+        if len(buckets_to_delete) > 0:
+            print("Attempting to delete buckets scheduled for deletion...")
+            delete_buckets(s3_client, buckets_to_delete)
+            print(Fore.GREEN + "Buckets deleted! Exiting script.")
+        else:
+            print(Fore.GREEN + "No buckets scheduled for deletion. Exciting script")
+        
+
+    except (ClientError, NoCredentialsError)  as e:
+        print(Fore.RED + f'Error connecting to S3! {e}')
+    
+
+def get_owner_email(s3_client, bucket_name):
+    '''
+    Gets the owner's email of an S3 bucket if that information is included in the bucket tags as OwnerEmail
+
+        Parameters:
+            s3_client (S3Client): the client necessary to execute S3 commands
+            bucket_name (str): the name of the bucket you would like the owner of
+        
+        Returns:
+            bucket_owner_email (str): Either the email of the owner of the bucket or No Owner if the no such email exists
+    '''
+    try:
+        get_bucket_tags_response = s3_client.get_bucket_tagging(
+            Bucket=bucket_name
+        )
+                    
+        bucket_owner_email_list = list(
+            filter(
+                lambda tag: tag['Key'] == "OwnerEmail",
+                get_bucket_tags_response['TagSet']
+            )
+        )
+
+        bucket_owner_email = "No Owner"
+        if len(bucket_owner_email_list) == 1:
+            bucket_owner_email = bucket_owner_email_list[0]['Value']
+
+        return bucket_owner_email
+
+    except s3_client.exceptions.from_code('NoSuchTagSetError') as e:
+        print(Fore.YELLOW + "Bucket does not have any tags!")
```

Comment:
I would still return some value else you will end up with a lot of `null` strings on the console/logs.